src/Calendar.php

```diff
@@ -270,7 +270,9 @@ public function toJsonLd(): array
             $jsonLd['endDate'] = $endDate->format(DateTime::ATOM);
         }
 
-        $jsonLd['status'] = $this->getStatusType()->toNative();
+        $jsonLd['status'] = [
+            'type' => $this->getStatusType()->toNative(),
+        ];
```

Comment:
Note that there already are some events/places that have this property in their JSON-LD projection. We can fix this by either replaying those events/places after this change is merged, or by adding a check on https://github.com/cultuurnet/udb3-silex/blob/master/src/Offer/ReadModel/JSONLD/NewPropertyPolyfillOfferRepository.php#L43-L45 to see if there's already a `status` and if it's a string move it to `status.type`. (You'll need to update that decorator anyway, to set the default value on `status.type` now instead of `status`)

Reply:
I created a ticket for this https://jira.uitdatabank.be/browse/III-3658